pymc3/sampling.py

```diff
@@ -81,7 +81,7 @@ def assign_step_methods(model, step=None, methods=(NUTS, HamiltonianMC, Metropol
     return steps
 
 
-def sample(draws, step=None, init='advi', n_init=200000, start=None,
+def sample(draws, step=None, n_advi=200000, n_nuts=2000, start=None,
```

Comment:
I think so. It limits what we can provide in terms of a docstring, but keeps things general. Another option would be a more explicit `init_params` argument that could vary depending on the init method chosen.